kele.el

```diff
@@ -642,9 +651,61 @@ not namespaced."
           ((&alist 'port port) (kele--ensure-proxy context))
           (url (format "http://localhost:%s/%s" port url-all)))
     (condition-case err
-        (plz 'get url :as #'json-read)
-      (error (signal 'kele-request-error (error-message-string err)))))) ;
-
+        (kele--resource-container-create
+         :resource (plz 'get url :as #'json-read)
+         :context context
+         :namespace namespace)
+      (error (signal 'kele-request-error (error-message-string err))))))
+
+;; TODO: add an option to filter out managed fields, similar to `kubectl get
+;; --show-managed-fields false' (the default behavior)
+;;
+;; This could even be a custom, e.g. `kele-filtered-object-fields'
```

Comment:
Ticket this up.

Reply:
```suggestion
;; TODO (#58): add an option to filter out managed fields, similar to `kubectl get
;; --show-managed-fields false' (the default behavior)
;;
;; This could even be a custom, e.g. `kele-filtered-object-fields'
```

See: #58.